designer/client/components/ComponentCreate/__tests__/__snapshots__/ComponentCreateList.jest.tsx.snap

```diff
@@ -233,6 +233,19 @@ exports[`ComponentCreateList should match snapshot 1`] = `
               A text box for users to enter multiple lines of text into, for example as 'Additional info'
             </div>
           </li>
+          <li>
```

Comment:
what is this file for?

Reply:
I generally advise against snapshots since they can end up being flakey/annoying to deal with/run slower but can have their uses

Reply:
ok, thanks, I'll come back to this part after in my PR :)